doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -4563,12 +4563,22 @@ Create Role
 
 Roles can be created globally (:ref:`create-global-role`) or for individual Dataverse collections (:ref:`create-role-in-collection`).
 
+.. _show-role:
+
 Show Role
 ~~~~~~~~~
 
-Shows the role with ``id``::
+You must have ``ManageDataversePermissions`` to be able to show a role that was created using :ref:`create-role-in-collection`. Global roles (:ref:`create-global-role`) only be shown with a superuser API token.
+
+A curl example using an ``ID``:
```

Comment:
Should we say that you may also use a role's alias?

Reply:
@sekmiller it didn't work for me. This is what I put in the description of this PR:

"Also, the "Show Role" code suggests that aliases are supported but they don't seem to work. So when I updated the docs I made sure to use an example with an ID."

Reply:
@sekmiller yes, as discussed in person. Good catch. Please see the commit message in d4f9189280 where I added this and made a few more improvements while I was in there.